Formula/ask-cli.rb

```diff
@@ -16,11 +16,16 @@ class AskCli < Formula
 
   def install
     system "npm", "install", *Language::Node.std_npm_install_args(libexec)
-    bin.install_symlink Dir["#{libexec}/bin/*"]
+    (bin/"ask").write <<~EOS
+      \#!/bin/bash
+      #{prefix}/libexec/bin/ask $@
+    EOS
+    bin.install_symlink
```

Comment:
Why didn't you use `write_exec_script` here? Also the parameterless `bin.install_symlink` looks superfluous to me.

```suggestion
    bin.write_exec_script libexec/"bin/ask"
```

Reply:
Honestly, I didn't know it existed. Thanks @chrmoritz !